Parser/FormTypeParser.php

```diff
@@ -253,8 +363,20 @@ private function parseForm($form)
 
                     if (($choices = $config->getOption('choices')) && is_array($choices) && count($choices)) {
                         $parameters[$name]['format'] = json_encode($choices);
-                    } elseif (($choiceList = $config->getOption('choice_list')) && $choiceList instanceof ChoiceListInterface) {
-                        $choices = $this->handleChoiceListValues($choiceList);
+                    } elseif ($choiceList = $config->getOption('choice_list')) {
+                        $choiceListType = $config->getType();
+                        $choiceListName = method_exists($choiceListType, 'getBlockPrefix') ?
+                            $choiceListType->getBlockPrefix() : $choiceListType->getName();
+
+
+                        if (('entity' === $choiceListName && false === $this->entityToChoice)) {
+                            $choices = array();
+                        } else {
+                            // TODO: fixme
```

Comment:
@bmxdasz i took it from https://github.com/nelmio/NelmioApiDocBundle/blob/master/Parser/FormTypeParser.php